### DISCUSSION TITLE
Discord login isn't saved in chrome profile.

### DISCUSSION THREAD

**Discussion by phantien02** [seleniumbase/SeleniumBase#2765] [Q&A] [answered]
Hi everyone. 
I'm trying to create chrome profile with Gmail,Twitter and Discord already logged in.
```
with SB(uc=True, user_data_dir=profile_path) as sb:
    # sb.sleep(10000)
    login_Gmail(sb)
    login_X(sb)
    login_Discord(sb)

```
This code creates a brand new chrome profile with custom user-data-dir and start logging in the Gmail, Twitter and Discord - all successfully log in.
But when I reopen the profile, only the Gmail and Twitter is already logged in and somehow the Discord is not saved - although the function is almost the same.
I would appreciate any help. Thank you so much!

**Comment by mdmintz** [comment, marked as answer]
Since it's working with Gmail and Twitter, but not Discord, I would assume that Discord is just being more strict about saved logins in Session Storage / Saved Cookies. There's probably not much else you can do from the SeleniumBase side, but you can explore around and maybe find a solution that works for you.
